### PR TITLE
Add ConfigConvert[Duration].

### PR DESCRIPTION
**Pull request by leifwickland**

### CODE REVIEW

File: src/main/scala/pureconfig/ConfigConvert.scala [hunk 1]
```diff
@@ -250,5 +250,23 @@ object ConfigConvert {
...
+}
 
+/**
+ * A ConfigConvert for Duration defined in a trait to give it lower priority.
+ */
+trait LowPriorityConfigConvertImplicits {
+  import scala.concurrent.duration.Duration
+  implicit val durationConfigConvert: ConfigConvert[Duration] = new ConfigConvert[Duration] {
+    override def from(config: RawConfig, namespace: String): Try[Duration] = {
+      config.get(namespace).fold[Try[Duration]](Failure(new Exception(s"Couldn't read config key $namespace."))) { stringyDuration =>
```
Comment (melrief) [inline src/main/scala/pureconfig/ConfigConvert.scala]: Any reason for `stringyDuration` instead of `stringDuration`?

**Comment by melrief** [comment]
Thanks for the changes @leifwickland, the `Duration` instance is really well done and the new low priority trait for `ConfigConvert` will help in future. LGTM if you address my small comment and then I'll release the new version of `pureconfig`.

File: src/main/scala/pureconfig/ConfigConvert.scala (hunk 1 above)
Comment (leifwickland) [inline src/main/scala/pureconfig/ConfigConvert.scala]: I was simply trying to use the typical English "<adjective> <noun>" construction. I've renamed all occurrences to `durationString`.

**Comment by leifwickland** [comment]
Thanks, Mario! Please let me know when you've published a new version.

**Comment by melrief** [comment]
Thanks to you Leif, nice work! I've just published 0.1.9.

**Pull request merged**